packages/context-menu/theme/lumo/vaadin-context-menu-styles.js

```diff
@@ -93,6 +93,10 @@ registerStyles(
     :host {
       user-select: none;
       -ms-user-select: none;
+    }
+
+    /* Workaround https://github.com/vaadin/web-components/issues/3133 */
+    :host(:hover) {
       -webkit-user-select: none;
```

Comment:
It's possible that once Safari starts supporting `user-select: none;` (without the prefix), this breaks without us knowing.

Reply:
Covered in https://github.com/vaadin/web-components/pull/3158/commits/ef27df0ca0c0a6a766ee47516d950bb468ab61df